### PR TITLE
Fix Qt5/4 import strategy.

### PR DESCRIPTION
**Pull request by jhoenicke**
- Try Qt5 before Qt4.
- Handle all exceptions (a ValueError is thrown if the wrong Qt was imported earlier)

### CODE REVIEW

**Comment by jhoenicke** [comment]
This fixes this bug with electrum:

```
  File "/home/hoenicke/.local/lib/python3.6/site-packages/trezorlib/qt/pinmatrix.py", line 21, in <module>
    from PyQt4.QtGui import (
ValueError: PyCapsule_GetPointer called with incorrect name
```

File: trezorlib/qt/pinmatrix.py [hunk 1]
```diff
@@ -18,34 +18,34 @@
...
         QVBoxLayout,
         QHBoxLayout,
     )
-    from PyQt4.QtCore import QObject, SIGNAL, QRegExp, Qt, QT_VERSION_STR
-except ImportError:
-    from PyQt5.QtWidgets import (
+    from PyQt5.QtGui import QRegExpValidator
+    from PyQt5.QtCore import QRegExp, Qt
+    from PyQt5.Qt import QT_VERSION_STR
+except Exception:
```
Comment (prusnak) [inline trezorlib/qt/pinmatrix.py]: Why the change from `ImportError` into `Exception`?

**Comment by matejcik** [comment]
looks good, thanks
